certified-connectors/Huddle US/apiDefinition.swagger.json

```diff
@@ -0,0 +1,896 @@
+          "type": "string",
+          "x-ms-summary": "Status"
+        },
+        "title": {
+          "description": "The task's title",
+          "type": "string",
+          "x-ms-summary": "Title"
+        },
+        "type": {
+          "description": "The type of task",
+          "type": "string",
+          "x-ms-summary": "Type"
+        },
+        "workspaceId": {
+          "description": "The ID of the workspace the task resides in",
+          "type": "string",
+          "x-ms-summary": "Workspace ID"
+        }
+      },
+      "type": "object"
+    },
+    "TaskInputs": {
+      "properties": {
+        "asignee": {
+          "description": "The ID of user to whom the task is assigned, if any",
+          "type": "string",
+          "x-ms-summary": "Asignee"
+        },
+        "description": {
+          "description": "A description of the task",
+          "type": "string",
+          "x-ms-summary": "Description"
+        },
+        "dueDate": {
+          "description": "The date by which the task should be completed",
+          "type": "string",
+          "x-ms-summary": "Due Date"
+        },
+        "fileId": {
+          "description": "The ID of the file the task is associated with, if any",
+          "type": "string",
+          "x-ms-summary": "File ID"
+        },
+        "id": {
+          "description": "The ID of the task",
+          "type": "string",
+          "x-ms-summary": "Task ID"
+        },
+        "status": {
+          "description": "The task's status",
+          "type": "string",
+          "x-ms-summary": "Status"
+        },
+        "title": {
+          "description": "The task's title",
+          "type": "string",
+          "x-ms-summary": "Title"
+        }
+      },
+      "type": "object"
+    },
+    "Workspace": {
+      "properties": {
+        "files": {
+          "description": "Files in the workspace's root directory",
+          "items": {
+            "type": "object"
+          },
+          "type": "array",
+          "x-ms-summary": "Files"
+        },
+        "folders": {
+          "description": "Folders in the workspace's root directory",
+          "items": {
+            "type": "object"
+          },
+          "type": "array",
+          "x-ms-summary": "Folders"
+        },
+        "id": {
+          "description": "The ID of the workspace",
+          "type": "string",
+          "x-ms-summary": "Workspace ID"
+        },
+        "title": {
+          "description": "The title of the workspace",
+          "type": "string",
+          "x-ms-summary": "Title"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "host": "msflow.huddle.com",
+  "info": {
+    "description": "**US GOVERNMENT & HEALTHCARE VERSION** (Standard version is also available.) Huddle is a secure document collaboration and client portal trusted by Enterprise and Government alike.",
+    "title": "Huddle for US Gov & Healthcare",
```

Comment:
Given the 30 character limit for titles, perhaps 'Huddle (US Gov & Healthcare)'? Or similar?

Reply:
Supposedly `Huddle for US Gov & Healthcare` is 30 characters... I didn't double check the automated validation, but my character count tool suggests that.

Are you seeing differently?

Reply:
@seanspeaks You're absolutely right, it's 30 char. Apologies - a space char made its way into my offline check right at the end, bringing it to 31. Disregard the above, we're good to go on the title.